backend/src/main/java/com/redhat/cloud/notifications/processors/email/EmailSender.java

```diff
@@ -110,6 +111,16 @@ public void init() {
                 emailTemplateService.renderTemplate(user, action, body)
         )
                 .asTuple()
+                .onFailure().invoke(templateEx -> {
+                    if (!(templateEx instanceof TemplateException) && !isInternalUser(user)) {
+                        logger.warnf(templateEx,
+                                "Unable to render template for bundle: [%s] application: [%s], eventType: [%s].",
+                                action.getBundle(),
+                                action.getApplication(),
+                                action.getEventType()
+                        );
+                    }
+                })
```

Comment:
We only need the old code here. There's no need for a an `instanceof TemplateException` check or the `isInternalUser` method. All exceptions will be logged at WARN level when an email is being sent (same behavior as before). Nothing will be logged when `EmailTemplateService` was called form the admin UI.

Reply:
Yes, I don't think the internal call reaches this point (no email is being sent).

Reply:
sorry.. i was a bit confused by this comment.. but I forgot to push my changes.